e2e/patient/patient-maintenance.e2e-spec.ts

```diff
@@ -0,0 +1,118 @@
+    import { PatientDetailPage } from './patient-detail/patient-dialog.po';
+    import { PatientAllergyDetailPage } from './patient-detail/patient-allergy-dialog.po';
+
+declare const allure: any;
+
+describe('TC0001_PatientManagement_e2e', () => {
+
+    const loginPage = new LoginPage();
+    const mainPage = new MainPage();
+    const patientMaintenancePage = new PatientMaintenancePage();
+    const allergyDetailPage = new AllergyDetailPage();
+    const patientDetailPage = new PatientDetailPage();
+    const patientAllergyDetailPage = new PatientAllergyDetailPage();
+
+    beforeAll(() => {
+        LoginNavigationService.navigateToHomePage(loginPage);
+        LoginNavigationService.login(loginPage);
+        MainNavigationService.navigateToPatientMaintenancePage(mainPage);
+    });
+
+    beforeEach(() => {
+        TestUtil.init('TC0001_PatientManagement_e2e', 'Purpose: This TC is intended to verify the CRUD of a Patient',
+            loginPage.appVersion, 'userName');
+    });
+
+    function createAllergyItem(allergyData: string[]) {
+        mainPage.getAllergyAddButton().click();
+        allergyDetailPage.checkPresentAndDisplayed();
+        allergyDetailPage.getNameInput().sendKeys(allergyData[0]);
+        allergyDetailPage.getSignsInput().sendKeys(allergyData[1]);
+        allergyDetailPage.getSymptomsInput().sendKeys(allergyData[2]);
+        allergyDetailPage.getButtonSubmit().click();
+        mainPage.checkConfSectionPresentAndDisplayed();
+        ExpectsUtil.checkCount(GridService.getGridInnerComponent(mainPage.getAllergyGrid()), 'Number of Allergies', 1);
+    }
+
+    function createPatientItem(patientData: string[]) {
+        patientMaintenancePage.getButtonAdd().click();
+        patientDetailPage.checkPresentAndDisplayed();
+        patientDetailPage.getNameInput().sendKeys(patientData[0]);
+        patientDetailPage.getSurnameInput().sendKeys(patientData[1]);
+        patientDetailPage.getEmailInput().sendKeys(patientData[2]);
+        patientDetailPage.getAddressStreetInput().sendKeys(patientData[3]);
+        patientDetailPage.getAddressCityInput().sendKeys(patientData[4]);
+        patientDetailPage.getAddressZipInput().sendKeys(patientData[5]);
+        patientDetailPage.getAddressCoordinatesInput().sendKeys(patientData[6])
+        patientDetailPage.getButtonSubmit().click();
+        patientMaintenancePage.checkPresentAndDisplayed();
+        ExpectsUtil.checkCount(GridService.getGridInnerComponent(patientMaintenancePage.getPatientsGrid()), 'Number of Patients', 1);
+    }
+
+    function assignAllegryToPatient(allergyNum: number) {
+        patientDetailPage.getAddButton().click();
+        patientAllergyDetailPage.checkPresentAndDisplayed();
+        patientAllergyDetailPage.getAllergyCombobox().click();
+        patientAllergyDetailPage.getAllergyList().get(0).click();
+        /*
+        patientAllergyDetailPage.getAssertedDateInput().sendKeys('01/01/2019'); // Error when sending these fields
```

Comment:
Here you have the steps to make it work:
1) Set an id to the systelab-datepicker
 <systelab-datepicker id="PatientAllergiesAssertedDateInput"...
2) Access method in the PO
	public getAssertedDateInput() {
		return this.getObjectById('PatientAllergiesAssertedDateInput').element(by.tagName('input'));
	}
3) patientAllergyDetailPage.getAssertedDateInput().sendKeys('01/01/2019'); will work